exporter/terra/terra_listener.py

```diff
@@ -73,8 +72,12 @@ def _experiment_message_handler(self, body: str, msg: Message):
         try:
             exp = ExperimentMessage.from_dict(json.loads(body))
             self.logger.info(f'Received experiment message for process {exp.process_uuid} (index {exp.experiment_index} for submission {exp.submission_uuid})')
+            start_time = datetime.datetime.now()
```

Comment:
A much cleaner and reusable approach would be to add a timing decorator to `terra_exporter.export()`